llama_stack/cli/download.py

```diff
@@ -405,8 +406,15 @@ def _meta_download(
     downloader = ParallelDownloader(max_concurrent_downloads=max_concurrent_downloads)
     asyncio.run(downloader.download_all(tasks))
 
-    print(f"\nSuccessfully downloaded model to {output_dir}")
-    cprint(f"\nMD5 Checksums are at: {output_dir / 'checklist.chk'}", "white")
+    cprint(f"\nSuccessfully downloaded model to {output_dir}", "green")
+    cprint(
+        f"\nView MD5 checksum files at following location: {output_dir / 'checklist.chk'}",
```

Comment:
I guess we could just say `files at` instead of `files at following location`

Reply:
Fixed.